AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -5,42 +5,41 @@
 //  Created by Lunar on 08/01/2021.
 //
 
-import SwiftUI
-import CoreLocation
-import CoreData
-import Charts
 import AirCastingStyling
+import Charts
+import CoreData
```

Comment:
I know it's been here, but just to point it out as the opportunity presented itself :) : such imports structure is already a pretty big [code smell](https://en.wikipedia.org/wiki/Code_smell) telling us that we probably have some serious architectural problems (no change required here ofc, just pointing out):
```Swift
import CoreData
import SwiftUI
```